drivers/serial/uart_nrfx_uarte.c

```diff
@@ -8,21 +8,24 @@
  * @brief Driver for Nordic Semiconductor nRF UARTE
  */
 
+#include <zephyr/kernel.h>
 #include <zephyr/drivers/uart.h>
+#include <zephyr/drivers/pinctrl.h>
 #include <zephyr/pm/device.h>
-#include <hal/nrf_uarte.h>
-#include <nrfx_timer.h>
 #include <zephyr/sys/util.h>
-#include <zephyr/kernel.h>
-#include <soc.h>
-#include <helpers/nrfx_gppi.h>
 #include <zephyr/linker/devicetree_regions.h>
 #include <zephyr/irq.h>
-
+#include <zephyr/cache.h>
+#include <soc.h>
+#include <hal/nrf_uarte.h>
+#include <nrfx_timer.h>
+#include <helpers/nrfx_gppi.h>
 #include <zephyr/logging/log.h>
-LOG_MODULE_REGISTER(uart_nrfx_uarte, CONFIG_UART_LOG_LEVEL);
+LOG_MODULE_REGISTER(uart_nrfx_uarte, 0);
 
-#include <zephyr/drivers/pinctrl.h>
+#define DMM_MEMORY_SECTION(...)
```

Comment:
This looks like a part of some other modification of the driver. Was the intent to replace `UARTE_MEMORY_SECTION()` with something else? In its current shape, it just removes assignments to certain linker sections for a few data structures. Or am I missing something?

Reply:
removed those changes.